glean-core/ios/Glean/Glean.swift

```diff
@@ -29,6 +29,8 @@ public class Glean {
         static let logTag = "glean/Glean"
     }
 
+    private let logger = Logger(tag: Constants.logTag)
```

Comment:
With just that one constant left, maybe it's enough to just inline it?

Reply:
I'm okay with that, I think I was mainly trying to keep up appearances.

Reply:
Hm, let's keep it consistent then and keep the constants. It's all gonna be compiled away